src/posts/en/configuring-babel-6-for-node-js.md

```diff
@@ -0,0 +1,175 @@
+console.log(jsRocks.next());
+
+```
+
+
+We'll install the **babel-cli** package by typing this command. This will install the latest stable version of **babel-cli** for the required project and list it as one of the `devDependencies` listing it in `package.json` too
+
+```bash
+npm install --save-dev babel-cli
+```
+
+Now if you run
+```bash
+babel code/index.js -d build/
+```
+
+#### Plugins and Presets
+
+You will see the same code that you wrote appear in `build/index.js`. This is where Babel **plugins** and **presets** come. Babel doesn't do much on it's own, but, with **plugins** and **presets** it can do a lot. We want all the ES2016 and ES2015 goodness in our code.
+
+In order to do that we'll install two presets as part of our devDependencies
+- [es2015](https://babeljs.io/docs/plugins/preset-es2015/)
+- [stage-0](https://babeljs.io/docs/plugins/preset-stage-0/)
+
+Type the following command in order to install these presets.
+```bash
+npm install --save-dev babel-preset-es2015 babel-preset-stage-0
+```
+In order to access a wide range of Babel plug ins click [here](https://babeljs.io/docs/plugins/)
+
+You need to include these presets in command you issue
+```bash
+babel --presets es2015,stage-0 code/index.js -o build/app.js
+```
+
+You will see normal es5 code generated in `app.js`, this is called **Transpiled code** (a term used widely in JS world). You can run this code using the command below.
+```bash
+node build/app.js
+```
+
+### Setting up a proper build environment using Babel
+This is all good magic but what about doing some proper development using node.js.
+
+#### babel configuration file .babelrc
+`.babelrc` is a very neat way to separate all your Babel stuff in one JSON file. It's also pretty easy to get started. This is our .babelrc file for this tutorial. You can access other .babelrc options [here](http://babeljs.io/docs/usage/options/) and make it as robust as you want it to be
+```json
+{
+  "plugins": ["es2015", "stage-0"]
+}
+```
+This is pretty much it of babel configuration for this tutorial. Now whenever we want to add or subtract plugins, instead of changing the command we will change the plugins array in this file. Easy! Isn't it?
+
+Now if you run
+```bash
+babel -w code/ -d build/
+```
+It will read the **presets** to use from `.babelrc` compile the code in `code/` directory and generate the compiled code javascript files in `build/` folder but hey! The command didn't end. Notice the `-w` flag, **w** stands for **watch**, it will recompile the code as you make changes in your code directory. COOL! Now this is some magic I am talking about.
+
+#### Using Source maps in your file
+If you are thinking that's all cool and fun but what about syntax highlighting and real time code debugging. You don't have to be worried. Source maps are just for this purpose. Source maps tell node.js that this code is transpiled and let's you highlight **source** file instead of the **transpiled** file 😄. YAYY!
```

Comment:
I guess source maps don't have much to do with "syntax highlighting" (when I read "syntax highlighting" I think about a text editor's syntax highlighting, not sure if this is what you mean).

> let's you highlight **source** file instead of the **transpiled** file

I guess there's a better verb than "highlight" here. Maybe "inspect" or "debug".